commons/zenoh-runtime/Cargo.toml

```diff
@@ -13,6 +13,10 @@ description = { workspace = true }
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+libc = { workspace = true }
+ron = { workspace = true }
+serde = { workspace = true }
+zenoh_runtime_derive = { path = "zenoh-runtime-derive" }
```

Comment:
Would the `path` directive work once published on `crates.io`? 
If we would need to publish the `zenoh-runtime-derive` crate, then the CI should be updated accordingly.

Reply:
I'm afraid that's correct. I prefer not to include them in the existing zenoh-macro to avoid polluting the library. What do you think? @Mallets

Reply:
I'm not sure to understand what's needed. Is publishing to `crates.io` needed or `path = "..."`?

Reply:
Needed to publish to crates.io.

Reply:
Then I would suggest to use `zenoh-macros` if possible. Adding a new crate only for this would require also to update the release CI. But first the crate should be reserved to `crates.io` and ownership shared with Eclipse.

Reply:
Okay. Let me handle this.

Reply:
Completed via db78a5a